convene-web/app/controllers/workspaces_controller.rb

```diff
@@ -0,0 +1,9 @@
+class WorkspacesController < ApplicationController
+  def show
+    @rooms = current_workspace.rooms
+  end
```

Comment:
I have a relatively strong preference to expose as few instance variables as possible in views. Could we instead expose the `workspace` and eager load the rooms; then rely on `workspace.rooms` to access the rooms in the UI?

This reduces coupling between the template layer and the controller layer.